ocaml/xapi/xapi_host.ml

```diff
@@ -2555,11 +2555,59 @@ let allocate_resources_for_vm ~__context ~self ~vm ~live =
   (* Implemented entirely in Message_forwarding *)
   ()
 
+(* Sync uefi certificates with the ones of the hosts *)
+let extract_certificate_file name =
+  if String.contains name '/' then
+    (* Internal error: tarfile not created correctly *)
+    failwith ("Invalid path in certificate tarball: " ^ name) ;
+  let path = Filename.concat !Xapi_globs.varstore_dir name in
+  debug "*** BRS: Extract UEFI cert %s" path ;
+  Helpers.touch_file path ;
+  path
+
+let with_temp_file_contents ~contents f =
+  let filename, out = Filename.open_temp_file "xapi_uefi_certificates" "tar" in
```

Comment:
It's more common to use a dash (`-`) in file names than the underscore.

Reply:
It comes from already existing code, but I can change that :+1: